src/reports/models/reportsManager.js

```diff
@@ -39,32 +45,49 @@ module.exports.getLastReports = async (limit) => {
 module.exports.postReport = async (testId, reportBody) => {
     const startTime = new Date(Number(reportBody.start_time));
     const job = await jobConnector.getJob(reportBody.job_id);
+    const phase = '0';
```

Comment:
do we support aggreation only with 1 phase? if yes lets open an issue in the project

Reply:
yes, added